index.html

```diff
@@ -29,29 +33,34 @@
         <!-- you can nest rows inside of columns! -->
         <!-- our board has 3 rows, each with 3 columns inside -->
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box box-1"></div>
+          <div class="col-xs-4 box box-2"></div>
+          <div class="col-xs-4 box box-3"></div>
         </div>
 
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box box-4"></div>
+          <div class="col-xs-4 box box-5"></div>
+          <div class="col-xs-4 box box-6"></div>
         </div>
 
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box box-7"></div>
+          <div class="col-xs-4 box box-8"></div>
+          <div class="col-xs-4 box box-9"></div>
         </div>
       </div>
     </div>
   </div>
-  
+
+    <div class="btn-group" role="group" aria-label="...">
+    <button type="button" class="btn btn-default reset-button">Reset</button>
+    </div>
+
   <!-- jquery -->
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
```

Comment:
Careful, this is jQuery 3 and not the 1.10 or 1.11 we're using elsewhere. Was this intentional?

It isn't a bad thing but can cause issues with some examples we use.